tests/Makefile

```diff
@@ -24,8 +26,16 @@ load-images: ## Load NGF and NGINX images on configured kind cluster
 test: ## Run the system tests against your default k8s cluster
 	go test -v ./suite -args --gateway-api-version=$(GW_API_VERSION) --image-tag=$(TAG) \
 		--ngf-image-repo=$(PREFIX) --nginx-image-repo=$(NGINX_PREFIX) --pull-policy=$(PULL_POLICY) \
-		--k8s-version=$(K8S_VERSION)
+		--k8s-version=$(K8S_VERSION) --service-type=$(GW_SERVICE_TYPE) --is-gke-internal-lb=$(GW_SVC_GKE_INTERNAL)
 
 .PHONY: delete-kind-cluster
 delete-kind-cluster: ## Delete kind cluster
 	kind delete cluster
+
+.PHONY: run-tests-on-vm
+run-tests-on-vm: ## Run the tests on a GCP VM
+	bash utils/run-tests-gcp-vm.sh
```

Comment:
Maybe just personal preference, but I like a `scripts` directory for these instead of `utils`.